platform/binance/api.go

```diff
@@ -198,6 +199,24 @@ func NormalizeToken(srcToken *Balance, tokens *TokenPage) (t blockatlas.Token, o
 		return blockatlas.Token{}, false
 	}
 
+	isFreeTokenNotZero, err := checkNotZeroValue(srcToken.Free)
+	if err != nil {
+		return blockatlas.Token{}, false
+	}
+	isFrozenTokenNotZero, err := checkNotZeroValue(srcToken.Frozen)
+	if err != nil {
+		return blockatlas.Token{}, false
```

Comment:
why we don't go to parse the token with the frozen balance is zero? can you give more description in this PR, please?

Reply:
We do not parse token if Frozen, Free and Locked balance is zero.
If at least 1 one of them is not zero - we will return it